src/NuGet.Core/NuGet.Commands/PublicAPI/netcoreapp5.0/PublicAPI.Shipped.txt

```diff
@@ -947,8 +947,6 @@ NuGet.Commands.VerifyArgs.LogLevel.get -> NuGet.Common.LogLevel
 NuGet.Commands.VerifyArgs.LogLevel.set -> void
 NuGet.Commands.VerifyArgs.Logger.get -> NuGet.Common.ILogger
 NuGet.Commands.VerifyArgs.Logger.set -> void
-NuGet.Commands.VerifyArgs.PackagePath.get -> string
-NuGet.Commands.VerifyArgs.PackagePath.set -> void
```

Comment:
This is a breaking change, so we probably shouldn't do it.

Reply:
[`VerifyArgs.PackagePath`](https://github.com/NuGet/NuGet.Client/pull/3662/files#diff-861924ebadd82aa092970c554f5ba0c3R28) property is now available only for `net472` and for other target frameworks we have `PackagesPath` property.